ONSdigital/rm-case-service
# Fix skipping of header row on CSV import

# Motivation and Context
The header row was being skipped when importing the CSV data export.

# What has changed
Fixed the script to no longer skip the header.

# How to test?
Run the steps per the instructions. Check the amount of rows inserted into the temporary table is equal to the number of lines in the CSV file.

# Links
Trello: https://trello.com/c/WNaIX2HO/295-add-survey-id-to-casegroups-l
